### PR TITLE
SLING-10836 - Update commons.threads to parent pom 45

### PR DESCRIPTION
**Pull request by rombert**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -105,4 +100,9 @@
             <version>1.2.6</version>
         </dependency>
     </dependencies>
+
+    <properties>
+        <sling.java.version>8</sling.java.version>
```
Comment (rombert) [inline pom.xml:101]: Technically speaking this is a change in requirements (used to be Java 7). We should probably bump the version to 3.3 after this.

File: pom.xml [hunk 2]
```diff
@@ -46,20 +45,8 @@
...
-                        <Bundle-Activator>
-                            org.apache.sling.commons.threads.impl.Activator
-                        </Bundle-Activator>
-                        <Provide-Capability>
-                            osgi.service;objectClass=org.apache.sling.commons.threads.ThreadPoolManager,
-                            osgi.service;objectClass=org.apache.sling.commons.threads.jmx.ThreadPoolMBean
-                        </Provide-Capability>
-                    </instructions>
-                </configuration>
+                <groupId>biz.aQute.bnd</groupId>
```
Comment (kwin) [inline pom.xml]: no need to duplicate this from parent: https://github.com/apache/sling-parent/blob/79c93908e42e6a3e09421c96ec2b96909741e891/sling-bundle-parent/pom.xml#L167. The relevant plugins are active by default.

File: pom.xml [hunk 3]
```diff
@@ -105,4 +100,9 @@
             <version>1.2.6</version>
         </dependency>
     </dependencies>
+
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
```
Comment (kwin) [inline pom.xml]: This does not work due to https://issues.apache.org/jira/browse/MJAVADOC-684. Instead use a timestamp like this: <project.build.outputTimestamp>2021-08-06T09:01:40Z</project.build.outputTimestamp>. The value does not matter as it is updated automatically during release:prepare.

**Review by kwin — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 3 above)
Comment (rombert) [reply pom.xml]: Fixed.

File: pom.xml (hunk 2 above)
Comment (rombert) [reply pom.xml]: Fixed.

**Comment by rombert** [comment]
Thanks for the review @kwin , can you please check again?

**Review by kwin — APPROVED** [review]

**Pull request merged**